lib/ret_web/router.ex

```diff
@@ -123,6 +127,11 @@ defmodule RetWeb.Router do
       resources("/slack", Api.V1.SlackController, only: [:create])
     end
 
+    scope "/v1", as: :api_v1 do
+      pipe_through([:portal_header_auth])
+      get("/presence", Api.V1.PresenceController, :show)
```

Comment:
We should probably put all these privileged portal apis under some sub-path `/api/v1/internal` or something, just so we can easily identify them and make sure to treat them specially. Maybe also reflect that in the name of the controller like `InternalPresenceController`.